src/main/java/org/example/pages/PuppySearchPage.java

```diff
@@ -0,0 +1,33 @@
+package org.example.pages;
+
+import org.openqa.selenium.By;
+
+import static org.example.core.DriverHolder.getDriver;
+
+public class PuppySearchPage {
+
+    public static final String PUPPY_URL = "http://localhost:8000/";
```

Comment:
Looks good, but I believe we don't need it anywhere else, so can be private

Reply:
In future we can use it for our tests. So, maybe it can be useful for as and he need make it public?.. Or... We can use it even if flag stay be private? Hm. Seems it was stupid question

You can criticize my english :D

Reply:
It's generally considered as a bad practice to keep unused code in the repo or open for other classes without a reason.